tutorial/debezium-with-oracle-jdbc/Dockerfile

```diff
@@ -1,14 +1,27 @@
 ARG DEBEZIUM_VERSION
 FROM quay.io/debezium/connect:$DEBEZIUM_VERSION
 ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc
-ENV INSTANT_CLIENT_DIR=/instant_client/
+
+# These define where we will automatically fetch the Oracle Instant Client from
+# This uses version 21.1.0.0, and all these should remain in-sync version wise with one another.
+ENV INSTANT_CLIENT_RPM_BASE_URL=https://yum.oracle.com/repo/OracleLinux/OL8/oracle/instantclient21/x86_64/getPackage/
```

Comment:
One question - would it be possible to add this as a repository using microdnf so it would be installed using the standard way?

Reply:
So you want to keep XStream in that example after all?